.changeset/friendly-deers-glow.md

```diff
@@ -0,0 +1,7 @@
+---
+"@fuel-ts/account": patch
+"fuels": patch
+"@fuel-ts/versions": patch
```

Comment:
Shouldn't this be breaking? 🤔 

At least at the CLI level, it will be because we're removing commands from it.